opentrons-ai-client/src/molecules/InputPrompt/index.tsx

```diff
@@ -285,22 +327,65 @@ export function InputPrompt(): JSX.Element {
   }, [data, isLoading, submitted])
 
   return (
-    <StyledForm id="User_Prompt">
-      <Flex css={CONTAINER_STYLE}>
-        <LegacyStyledTextarea
-          rows={calcTextAreaHeight(watchUserPrompt)}
-          placeholder={t('type_your_prompt')}
-          {...register('userPrompt')}
-        />
-        <SendButton
-          disabled={watchUserPrompt.length === 0}
-          isLoading={isLoading}
-          handleClick={() => {
-            handleClick()
-          }}
-        />
-      </Flex>
-    </StyledForm>
+    <form id="User_Prompt" className={styles.form}>
+      {/* Error message */}
+      {fileError && (
+        <div className={styles.error_container}>
+          <StyledText color={COLORS.red50} fontSize={TYPOGRAPHY.fontSizeH3}>
+            {fileError}
+          </StyledText>
+        </div>
+      )}
+
+      {/* Main input container */}
+      <div className={styles.main_input_container}>
+        {/* Display attached files above the input */}
+        {attachedFiles.length > 0 && (
+          <div className={styles.attached_files_section}>
+            <div className={styles.attached_files_list}>
+              {attachedFiles.map((file, index) => (
+                <AttachedFileItem
+                  key={`${file.name}-${index}`}
+                  file={file}
+                  onRemove={() => {
+                    handleRemoveFile(index)
+                  }}
+                  showRemoveButton={true}
+                />
+              ))}
+            </div>
+          </div>
+        )}
+
+        {/* Text input area - separate row */}
+        <div className={styles.text_input_section}>
+          <textarea
+            rows={calcTextAreaHeight(watchUserPrompt)}
+            placeholder={t('type_your_prompt')}
+            className={styles.textarea}
+            {...register('userPrompt')}
+          />
+        </div>
+
+        {/* Bottom row with attach button and send button */}
+        <div className={styles.button_row_container}>
+          <AttachFileButton
+            onFileSelect={handleFileSelect}
+            disabled={
+              isLoading || attachedFiles.length >= MAX_FILES_PER_MESSAGE
+            }
+          />
+          <div className={styles.spacer} />
+          <SendButton
+            disabled={watchUserPrompt.length === 0}
+            isLoading={isLoading}
+            handleClick={() => {
+              handleClick()
+            }}
```

Comment:
not work?
```suggestion
            handleClick={handleClick}
```

i think the prop name I named isn't good. i will open a pr to mofity that.